src/hash_test.cpp

```diff
@@ -3,73 +3,253 @@
 /* Include simhash */
 #include "hash.hpp"
 
-static int bit_count(uint64_t value) {
-    int count = 0;
-    while (value) {
-        value &= value - 1;
-        ++count;
+#define vector_initializer(data) (data), \
```

Comment:
If you're going to use macros, make them SCREAMING_SNAKE_CASE to display the monstrosity that they truly are.